### PR TITLE
Changes to reconciliations job on NSF

### PR DESCRIPTION
**Pull request by sumesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/4854

*Description of changes:*
- Changes to reconciliations job on NSF

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updated the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: queue_services/payment-reconciliations/tests/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+http {
+
+  server {
+    listen 80;
+
+    location /paybc-api {
+    	if ($request_uri ~* "([^/]*$)" ) {
+		  set  $last_path_component  $1;
```
Comment (pwei1018) [inline queue_services/payment-reconciliations/tests/nginx.conf:14]: It seems here has formatting issue.

File: queue_services/payment-reconciliations/src/reconciliations/worker.py [hunk 2]
```diff
@@ -134,8 +140,8 @@ async def _update_payment_details(msg: Dict[str, any]):
                 # NSF Condition. Publish to account events for NSF.
                 _process_failed_payments(payment, row)
                 # Send mailer and account events to update status and send email notification
-                await _publish_account_events('PaymentFailed', payment_account, row)
-                await _publish_mailer_events('PaymentFailed', payment_account, row)
+                await _publish_account_events('lockAccount', payment_account, row)
+                await _publish_mailer_events('lockAccount', payment_account, row)
```
Comment (saravanpa-aot) [inline queue_services/payment-reconciliations/src/reconciliations/worker.py:144]: we can think of moving this to account -events as well.. just to be in sync ..incase if mail goes before auth-db is updated

**Review by saravanpa-aot — APPROVED** [review]
was looking for the place for lock_accounts..but cudnt find any ..But i guess thats already handled in the code

File: queue_services/payment-reconciliations/tests/nginx.conf (hunk 1 above)
Comment (sumesh-aot) [reply queue_services/payment-reconciliations/tests/nginx.conf]: Done

File: queue_services/payment-reconciliations/src/reconciliations/worker.py (hunk 2 above)
Comment (sumesh-aot) [reply queue_services/payment-reconciliations/src/reconciliations/worker.py]: Done

**Pull request merged**